common/content/services.js

```diff
@@ -16,7 +16,7 @@ const Services = Module("services", {
         this.jsm = window.Services;
         this.services = {
             "autoCompleteSearch": {
-                class_: "@mozilla.org/autocomplete/search;1?name=history",
+                class_: "@mozilla.org/autocomplete/search;1?name=" + (services.get("vc").compare(VERSION, "49") < 0 ? "history" : "unifiedcomplete"),
```

Comment:
@grassofhust 

Was `49.0` and now `49.0.1`. But `VERSION` is not the problem. Even when I hard-coded `"49.0.1"`, Vimperator was still broken.

What actually worked is replacing `services.get("vc")` with `this.jsm["vc"]`.